config/config.go

```diff
@@ -1561,10 +1561,9 @@ func (cfg *CGRConfig) V1ReloadConfig(args *ConfigReloadArgs, reply *string) (err
 	} else {
 		err = cfg.reloadSections(args.Section)
 	}
-	if err != nil {
-		return
+	if err == nil {
```

Comment:
Please keep here the old approach with checking the error first since it is  more clear that we treat it.